src/library_pthread.js

```diff
@@ -267,11 +273,15 @@ var LibraryPThread = {
           cancelThread(d['thread']);
         } else if (cmd === 'loaded') {
           worker.loaded = true;
+#if ENVIRONMENT_MAY_BE_NODE
```

Comment:
Can you add a comment here (or at least in one of the places we call ref/unref) describing why we do it?

Reply:
Sure.